content/SCALE/SCALETutorials/SystemSettings/Services/S3ServiceSCALE.md

```diff
@@ -27,29 +28,24 @@ Because S3 is the de facto standard for cloud-based storage, setting up an S3 se
 Having large numbers of files (>100K for instance) in a single bucket with no sub-directories can harm performance and cause stability issues.
 {{< /hint >}}
 
-Go to the **System Settings > Services** and find **S3**, then click <i class="material-icons" aria-hidden="true" title="Configure">edit</i> to configure the service.
-
-![ServicesS3SCALE](/images/SCALE/ServicesS3SCALE.png "S3 Service Options")
+Go to the **System Settings > Services** and find **S3**, then click <i class="material-icons" aria-hidden="true" title="Configure">edit</i> to open the **Services > S3** screen to configure the service.
 
-{{< expand "Field Descriptions" "v" >}}
-{{< include file="/_includes/ServicesS3Fields.md" type="page" >}}
-{{< /expand >}}
+![S3ServiceSettings(/images/SCALE/22.02/S3ServiceSettings.png "S3 Service Options")
 
-Select a clean dataset, one that doesn't have existing data files.
-Minio manages files as objects that you *cannot* mix with other dataset files.
-You can create new datasets by going to **Storage** and clicking <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> > **Add Dataset**.
+First, select a clean dataset, one that does not have existing data files. If you do not have a clean dataset, [create a dataset]({{< relref "DatasetSCALE.md" >}}).
```

Comment:
Build test found an error (**Page not found**) with this reference:
![image](https://user-images.githubusercontent.com/17933320/191567907-df627b51-c636-4df3-a6e5-f17ad2342062.png)

Reply:
fixed link with a typo in the file name, datasetsSCALE.md is the correct file name.